From 9a996e048362245002b55dc52d954a8282a7e228 Mon Sep 17 00:00:00 2001
From: Aaron Kling <webgeek1234@gmail.com>
Date: Mon, 13 May 2024 13:02:27 -0500
Subject: [PATCH 4/7] Lower q/android-4.9 requirements

Since p/android-4.9 is not available

* Lower minlts to 4.9.141
* Remove xfrm interface as it's not available in 4.9.141
* Remove usb dependency on rtl8152

Change-Id: Ie60e8ddd82142d869664498185707b7bb3593066
---
 q/android-4.9/android-base-conditional.xml | 16 +---------------
 q/android-4.9/android-base.config          |  1 -
 2 files changed, 1 insertion(+), 16 deletions(-)

diff --git a/q/android-4.9/android-base-conditional.xml b/q/android-4.9/android-base-conditional.xml
index 0ba7e5b..ba664a7 100644
--- a/q/android-4.9/android-base-conditional.xml
+++ b/q/android-4.9/android-base-conditional.xml
@@ -1,4 +1,4 @@
-<kernel minlts="4.9.165" />
+<kernel minlts="4.9.141" />
 
 <!-- KEEP ALPHABETICALLY SORTED -->
 <!-- ARM base requirements -->
@@ -143,17 +143,3 @@
 	</config>
 </group>
 
-<!-- CONFIG_USB implies CONFIG_USB_RTL8152 -->
-<group>
-	<conditions>
-		<config>
-			<key>CONFIG_USB_RTL8152</key>
-			<value type="bool">n</value>
-		</config>
-	</conditions>
-	<config>
-		<key>CONFIG_USB</key>
-		<value type="bool">n</value>
-	</config>
-</group>
-
diff --git a/q/android-4.9/android-base.config b/q/android-4.9/android-base.config
index 795cde5..136c2c2 100644
--- a/q/android-4.9/android-base.config
+++ b/q/android-4.9/android-base.config
@@ -218,6 +218,5 @@ CONFIG_USB_CONFIGFS_F_MIDI=y
 CONFIG_USB_CONFIGFS_UEVENT=y
 CONFIG_USB_GADGET=y
 CONFIG_USB_SUPPORT=y
-CONFIG_XFRM_INTERFACE=y
 CONFIG_XFRM_STATISTICS=y
 CONFIG_XFRM_USER=y
-- 
2.46.0

